synapse/models/infotech.py

```diff
@@ -1076,20 +1083,46 @@ async def _onFormItDevStr(node):
                 'doc': 'The software creates the file path.'}),
 
             (('it:software', 'creates', 'it:os:windows:registry:entry'), {
-                'doc': 'The software creates the windows registry entry.'}),
+                'doc': 'The software creates the Microsoft Windows registry entry.'}),
+
+            (('it:software', 'creates', 'it:os:windows:service'), {
+                'doc': 'The software creates the Microsoft Windows service.'}),
 
             (('it:exec:query', 'found', None), {
                 'doc': 'The target node was returned as a result of running the query.'}),
 
-            # FIXME tighten these down...
-            (('it:app:snort:rule', 'detects', None), {
-                'doc': 'The snort rule is intended for use in detecting the target node.'}),
+            (('it:app:snort:rule', 'detects', 'risk:vuln'), {
+                'doc': 'The snort rule detects use of the vulnerability.'}),
+
+            (('it:app:snort:rule', 'detects', 'it:software'), {
+                'doc': 'The snort rule detects use of the software.'}),
+
+            (('it:app:snort:rule', 'detects', 'risk:tool:software'), {
+                'doc': 'The snort rule detects use of the tool.'}),
+
+            (('it:app:snort:rule', 'detects', 'ou:technique'), {
+                'doc': 'The snort rule detects use of the technique.'}),
 
-            (('it:app:yara:rule', 'detects', None), {
-                'doc': 'The YARA rule is intended for use in detecting the target node.'}),
```

Comment:
Same comment as above - have seen (arguably dumb) YARA rules to detect individual "blacklisted" files.